handlers.go

```diff
@@ -2881,10 +2882,13 @@ func (s *Server) handleRegisterUser(c echo.Context) error {
 		}
 	}
 
+	token := "EST" + uuid.New().String() + "ARY"
 	authToken := &AuthToken{
```

Comment:
I think we have to stop creating API keys on user registration, and instead let the users create a key after registering. It will improve the UX and allow them properly label their keys

Reply:
i think this is necessary since the key management endpoints require a bearer token. this seems to give the newly registered user a token for their browser session upon successful register.

on cleanliness - it seems we could reduce code duplication here and have all new tokens be created the newAuthTokenForUser method. happy to do that cleanup in a separate PR